<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPatient</class>
 <widget class="QDialog" name="EditPatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>848</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>323</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="patientInfoLayout">
    <item>
     <widget class="QLabel" name="patientInformation">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Patient Information&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="labelLayout">
        <item>
         <widget class="QLabel" name="patientNameLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Patient&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="firstNameLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;First Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lastNameLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Last Name&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ageLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Age&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="phoneNumberLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Phone Number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="genderLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Gender&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dobLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;DOB&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ssNumLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Social Security Number&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="doctorLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Doctor&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="inputsLayout">
        <item>
         <widget class="QComboBox" name="patientComboBox"/>
        </item>
        <item>
         <widget class="QLineEdit" name="firstNameEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lastNameEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="ageInput"/>
        </item>
        <item>
         <widget class="QLineEdit" name="phoneNumberEdit"/>
        </item>
        <item>
         <widget class="QComboBox" name="genderComboBox">
          <item>
           <property name="text">
            <string>Male</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Female</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dobEdit">
          <property name="displayFormat">
           <string>yyyy-MM-dd</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="socialSecurityNumberInput"/>
        </item>
        <item>
         <widget class="QComboBox" name="doctorComboBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="updateButton">
      <property name="text">
       <string>Update</string>
      </property>
      <property name="icon">
       <iconset resource="img.qrc">
        <normaloff>:/img/img/correct.png</normaloff>:/img/img/correct.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="scheduleView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>601</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>431</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainSceduleLayout">
    <item>
     <widget class="QLabel" name="scheduleTitle">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Add to Pill Schedule&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="subSceduleLayout">
      <item>
       <layout class="QVBoxLayout" name="scheduleLabels">
        <item>
         <widget class="QLabel" name="assignedNurseLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Assigned Nurse&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="medLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Medicine&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dosageLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Dosage&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="intervalLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Intervals&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dateBeginLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Begin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="dateEndLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;End&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="sceduleInputsLayout">
        <item>
         <widget class="QComboBox" name="assignedNurseComboBox"/>
        </item>
        <item>
         <widget class="QComboBox" name="medComboBox"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="dosageInputLayout">
          <item>
           <widget class="QLineEdit" name="dosageValueInNum">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dosageValueInUnits">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="intervalLayout">
          <item>
           <widget class="QLineEdit" name="intervaValueNum"/>
          </item>
          <item>
           <widget class="QComboBox" name="intervalValueComboBox">
            <item>
             <property name="text">
              <string>Hours</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Days</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Mintues</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Seconds</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QDateEdit" name="dateBeginInput">
          <property name="displayFormat">
           <string>yyyy-MM-dd</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEndInput">
          <property name="displayFormat">
           <string>yyyy-MM-dd</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="scheduleButtons">
      <item>
       <widget class="QPushButton" name="addToSchedule">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/img/plus.png</normaloff>:/img/img/plus.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateScheduleButton">
        <property name="text">
         <string>Update</string>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/img/correct.png</normaloff>:/img/img/correct.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeFromSchedule">
        <property name="text">
         <string>Remove</string>
        </property>
        <property name="icon">
         <iconset resource="img.qrc">
          <normaloff>:/img/img/minus.png</normaloff>:/img/img/minus.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="idScheduleView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>101</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="userpic">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>-50</y>
     <width>881</width>
     <height>571</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="img.qrc">:/img/img/background.png</pixmap>
   </property>
  </widget>
  <zorder>userpic</zorder>
  <zorder>idScheduleView</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>scheduleView</zorder>
  <zorder>layoutWidget</zorder>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
